python/tvm/contrib/torch/optimize_torch.py

```diff
@@ -51,14 +51,6 @@ def forward(self, *torch_inputs: Tuple[torch.Tensor]):
         return ret
 
 
-@register_func("script_torch.save_to_base64")
```

Comment:
let us keep torch related functionalities untouched for now, since it would be good to first focus on runtime module serialization mechanism that **do not** handle shared libraries, but mainly binary exportables.

Reply:
Thanks for the feedback. They are moved back to torch related functionalities while keeping the current changes (e.g., remove the unrecommended use of `tmpnam`, keep the better naming).